upstream: yes, master branch

From 09408f9ae3e4169d4810cfe6d5ed9d6bcea75761 Mon Sep 17 00:00:00 2001
From: Michael Jansen <kde@michael-jansen.biz>
Date: Wed, 16 Oct 2013 23:26:50 +0200
Subject: [PATCH] FFMPEG removed the deprecated AVCODEC_MAX_AUDIO_FRAME_SIZE
 define

Found the value on the web. Where is given as a comment in the change.
---
 plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp b/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp
index 8bae68a..c4a63f0 100644
--- a/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp
+++ b/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp
@@ -65,6 +65,13 @@ extern "C" {
 #  define CODEC_ID_MP3 CODEC_ID_MP3LAME
 #endif
 
+// Has been deprecated for ages. Now removed. For now do what ffmpeg does
+// http://ffmpeg.org/pipermail/ffmpeg-cvslog/2012-August/053785.html
+// 192000 = 1 second of 48khz 32bit audio
+#ifndef AVCODEC_MAX_AUDIO_FRAME_SIZE
+#define AVCODEC_MAX_AUDIO_FRAME_SIZE 192000
+#endif
+
 K3bFFMpegWrapper* K3bFFMpegWrapper::s_instance = 0;
 
 
-- 
1.8.4.4

